hs_core/management/commands/migrate_raster_resources.py

```diff
@@ -0,0 +1,160 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from hs_core.hydroshare import current_site_url, set_dirty_bag_flag
+from hs_core.models import CoreMetaData, Coverage
+from hs_geo_raster_resource.models import RasterResource
+from hs_file_types.models import GeoRasterLogicalFile
+from ..utils import migrate_core_meta_elements
+
+
+class Command(BaseCommand):
+    help = "Convert all raster resources to composite resource"
+
+    def handle(self, *args, **options):
+        logger = logging.getLogger(__name__)
+        resource_counter = 0
+        create_raster_aggregation = False
+        to_resource_type = 'CompositeResource'
+        msg = "THERE ARE CURRENTLY {} RASTER RESOURCES PRIOR TO CONVERSION.".format(
+            RasterResource.objects.all().count())
+        logger.info(msg)
+        print(">> {}".format(msg))
+
+        for rast_res in RasterResource.objects.all():
+            # check resource exists on irods
+            istorage = rast_res.get_irods_storage()
+            if not istorage.exists(rast_res.root_path):
+                err_msg = "Raster resource not found in irods (ID: {})".format(rast_res.short_id)
+                logger.error(err_msg)
+                print("Error:>> {}".format(err_msg))
+                # skip this raster resource
+                continue
+
+            if rast_res.metadata.cellInformation is not None:
+                # get the vrt file name which needs to be used to create a new folder for
+                # raster aggregation
+                vrt_file = None
+                for res_file in rast_res.files.all():
+                    if res_file.extension.lower() == '.vrt':
+                        vrt_file = res_file
+                        break
+
+                create_raster_aggregation = vrt_file is not None
```

Comment:
Here's the problem.  If the rater resource doesn't have metadata celllinformation it will keep the state of the previous resource.  I'll submit a patch